app/pages/reporter/application/[applicationId]/index.tsx

```diff
@@ -1,71 +1,92 @@
 import React, {Component} from 'react';
 import {graphql} from 'react-relay';
-import {applicationQueryResponse} from 'applicationQuery.graphql';
+import {ApplicationIdPageQueryResponse} from 'ApplicationIdPageQuery.graphql';
 import {CiipPageComponentProps} from 'next-env';
 import DefaultLayout from 'layouts/default-layout';
 import ApplicationWizard from 'containers/Applications/ApplicationWizard';
 import {USER} from 'data/group-constants';
 import {NextRouter} from 'next/router';
+import ViewApplication from './version/[versionNumber]/view';
 
 const ALLOWED_GROUPS = [USER];
 
 interface Props extends CiipPageComponentProps {
-  query: applicationQueryResponse['query'];
+  query: ApplicationIdPageQueryResponse['query'];
   router: NextRouter;
 }
-class Application extends Component<Props> {
+class ApplicationPage extends Component<Props> {
   static allowedGroups = ALLOWED_GROUPS;
   static isAccessProtected = true;
   static query = graphql`
-    query applicationQuery(
-      $formResultId: ID!
-      $applicationId: ID!
-      $version: String!
-    ) {
+    query ApplicationIdPageQuery($applicationId: ID!) {
       query {
         application(id: $applicationId) {
           id
           rowId
           latestDraftRevision {
+            ...ApplicationWizard_applicationRevision
             versionNumber
             legalDisclaimerAccepted
           }
           facilityByFacilityId {
             bcghgid
           }
+          latestSubmittedRevision {
+            versionNumber
+          }
         }
         session {
           ...defaultLayout_session
         }
         ...ApplicationWizard_query
-          @arguments(
-            formResultId: $formResultId
-            applicationId: $applicationId
-            version: $version
-          )
       }
     }
   `;
 
   static getInitialProps = () => ({
     variables: {
-      formResultId: '',
-      applicationId: '',
-      version: ''
+      applicationId: ''
+    }
+  });
+
+  static getRoute = (
+    applicationId: string,
+    formId?: string,
+    confirmationPage = false
+  ) => ({
+    pathname: '/reporter/application/[applicationId]/',
+    query: {
+      applicationId,
+      formId,
+      confirmationPage
     }
   });
 
   render() {
     const {query, router} = this.props;
     const {session} = query || {};
     const {application} = query || {};
-
+    console.log(query);
```

Comment:
remove console.log()
Everything else looks good, so I can remove this in my PR once it is rebased instead of waiting for CI again for this very minor change.